GIT2/src/main/java/FizzBuzz.java

```diff
@@ -0,0 +1,25 @@
+public class FizzBuzz {
+    private String Test;
+    private Integer integer;
+
+    public FizzBuzz(String test, Integer integer) {
+        Test = test;
+        this.integer = integer;
+    }
+
+    public String getTest() {
```

Comment:
Gettrs and setters are too difficult, get rid of them!!!!!

Reply:
Done, hope youre happy